tutorials/kernelshap_tabular_penguin.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>I don't understand line 11. '''pred_onnx''' doesn't do anything and I'm actually a bit surprised that is is even valid code. I guess you're allowed to write whatever you like in Python ;-)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Well spot. That's a typo I guess. I simply copy paste this part from other notebooks. I will fix those. Thanks!</p>